gateway/platforms/vertx3/vertx3/src/main/java/io/apiman/gateway/platforms/vertx3/engine/VertxPluginRegistry.java

```diff
@@ -62,8 +57,34 @@ private static File getTempPluginsDir() {
      * @param config the plugin config
      */
     public VertxPluginRegistry(Vertx vertx, VertxEngineConfig vxEngineConfig, Map<String, String> config) {
-        super(getTempPluginsDir(), PluginUtils.getDefaultMavenRepositories());
-        this.client = vertx.createHttpClient();
+
+        super(config);
+
+        this.vertx=vertx;
+
+        //Get HTTPS Proxy settings (useful for local dev tests and corporate CI)
+        String sslProxyHost = System.getProperty("https.proxyHost", "none");
```

Comment:
Interesting idea!

Reply:
And useful when internet outbound is protected by a corporate proxy.